styles.css

```diff
@@ -416,3 +442,913 @@ textarea {
+  }
+
+  .deskcontact :nth-child(1) {
+    margin-right: auto;
+  }
+
+  .deskcontact input {
+    min-width: 45%;
+  }
+
+  form button {
+    margin-left: 0;
+    padding: 1%;
+    width: 25%;
+    margin-bottom: 5%;
+  }
+
+  .hra {
+    margin-top: 20%;
+  }
+
+  .languagescenter {
+    display: flex;
+    justify-content: space-evenly;
+    text-align: left;
+  }
+
+  header a {
+    color: #fff;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .frontpage {
+    margin-bottom: 10%;
+  }
+
+  main {
+    background-color: #1c1a19;
+    background-image: url(./images/desky/group.svg);
+    background-size: 45%;
+    background-repeat: no-repeat;
+    background-position: top 0% right;
+    padding-top: 5%;
+    display: flex;
+    flex-direction: row-reverse;
+    align-items: baseline;
+    justify-content: flex-end;
+    padding-bottom: 2%;
+  }
+
+  .bars {
+    display: none;
+  }
+
+  .logo {
+    margin-left: 10%;
+  }
+
+  header {
+    background-color: #3c3a39;
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    padding-bottom: 1%;
+    padding-top: 1%;
+  }
+
+  .frontpage ul {
+    flex-direction: column;
+  }
+
+  main section {
+    width: 50%;
+  }
+
+  .recent {
+    color: #1c1a19;
+    line-height: 52px;
+    border-bottom: none;
+    text-align: left;
+    padding: 0;
+    margin: 0;
+  }
+
+  .line {
+    display: flex;
+    gap: 2%;
+    align-items: center;
+    justify-content: flex-start;
+    padding: 5% 5%;
+  }
+
+  .line img {
+    max-width: 50%;
+  }
+
+  .card img {
+    width: 100%;
+  }
+
+  .projimage {
+    display: inline;
+    float: left;
+    width: 90%;
+  }
+
+  .card button {
+    padding: 1%;
+    background-color: #ff6b00;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    color: #fff;
+    border: none;
+    width: 40%;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+    gap: 2%;
+    margin: 0;
+    padding: 0;
+    width: 80%;
+  }
+
+  .text {
+    max-width: 40%;
+    padding-right: 0;
+  }
+
+  .cardheader {
+    margin-top: 0;
+  }
+
+  .card li {
+    padding: 2%;
+  }
+
+  .grids {
+    width: 80%;
+    display: flex;
+    gap: 4%;
+    flex-wrap: wrap;
+    margin-top: 5%;
+    padding-bottom: 0;
+  }
+
+  .card2 {
+    background-image: url(./images/projects/project\ background.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 30%;
+    color: #fff;
+    padding-top: 0;
+    margin-top: 0;
+    margin-left: 0;
+    margin-right: 0;
+    margin-bottom: 0;
+    padding-bottom: -90%;
+    position: relative;
+    min-height: 500px;
+  }
+
+  .card2 p {
+    font-size: 17px;
+    margin-top: 0;
+    width: 100%;
+    position: absolute;
+    bottom: 15%;
+  }
+
+  .card2 h2 {
+    font-size: 32px;
+    margin-bottom: 0;
+    position: absolute;
+    bottom: 50%;
+  }
+
```

Comment:
- [ ] The previous reviewer suggested you to apply the hover design in the cards. right now the first card has `see project` button but you have placed it statically so it remains there in the card. what he meant was and the design requirement is that when user hovers on any project card, the `see project` button should appear and also border should appear around the card. 
you can style it using `:hover` pseudo-class on the button and implement these 2 required changes.
![c](https://github.com/mj-e-boc/M-J-Portfolio/assets/96683843/b9b8569b-0ee5-4fdc-8d01-0967937f2e05)


- [ ] Kindly consider applying button interactions styling in the hover and pressed states as requested by the previous reviewer because I can see that you have not applied this in any of your button. you can apply this using :active and :hover pseudo-classes following the figma design guidelines. your buttons should change color when hovered or pressed.
design guidelines
![x](https://github.com/mj-e-boc/M-J-Portfolio/assets/96683843/1e39afbe-5677-4421-980e-ef0a2bd88ff6)